pkg/apiserver/authorization/authorizer/interfaces.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2020 The KubeSphere Authors.
```

Comment:
Is this file copied from K8s? If so, we can't claim the copyright. I think it should copy the file as it is including the license info.

Reply:
> if you copy AL-licensed source code verbatim, you should keep that license on that source code. if you make a derived work, you can change the license, but you have to comply with the terms of the license.
https://opensource.stackexchange.com/questions/1713/adding-feature-to-an-apache-v2-package